data/images/objects/bonus_block/heavy-brick.sprite

```diff
@@ -0,0 +1,42 @@
+(supertux-sprite
+      (action
```

Comment:
Make indentation consistent please.

Reply:
Should be fixed now with the latest commit.